open_academy/models/session.py

```diff
@@ -1,14 +1,43 @@
-from odoo import models, fields
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
 
 
 class Session(models.Model):
     _name = "session"
     _description = "Session"
 
     name = fields.Char(required=True)
-    start_date = fields.Date()
+    start_date = fields.Date(default=lambda self: fields.Date.today(self))
     duration = fields.Float()
     seats = fields.Integer(string="Number of seats")
+    taken_seats = fields.Integer()
+    percent_seats = fields.Integer(string="Percentage of taken seats", compute="_compute_seats", default=0)
+    active = fields.Boolean(default=True)
     instructor_id = fields.Many2one("res.partner", domain="[('instructor', '=', True)]")
     course_id = fields.Many2one("course")
     attendee_ids = fields.Many2many("res.partner")
+
+    @api.depends("seats", "taken_seats")
+    def _compute_seats(self):
+        for record in self:
+            if record.seats > 0:
+                record.percent_seats = round(record.taken_seats / record.seats * 100)
```

Comment:
Dude.... Now you miss compute `seats` and `taken_seats`

Reply:
@EmilioSerna Before the next revision, perform functional flow tests and make sure that everything is working properly.

Reply:
Done